app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.example.bigowlapp">
 
     <uses-permission android:name="android.permission.SEND_SMS" />
     <uses-permission android:name="android.permission.READ_CONTACTS" />
+    <uses-permission android:name="android.permission.READ_PRIVILEGED_PHONE_STATE"
+        tools:ignore="ProtectedPermissions" />
```

Comment:
Can this permission be removed, because from what I looked into, this permission would only be allowed for Android System Apps only, which would suggest it wouldn't be allowed for regular apps like ours.
https://developer.android.com/training/articles/user-data-ids#:~:text=Android%2010%20(API%20level%2029,order%20to%20access%20these%20identifiers.

Maybe using a solution here would make more sense for device identification: 
- https://developer.android.com/training/articles/user-data-ids (androids recommendations)
- https://stackoverflow.com/questions/58022573/in-android-10-getdeviceid-value-is-null (something I found on Stackoverflow)

Reply:
Yes, I can remove that. I just forgot to do that earlier

Reply:
I have decided to use the firebase identifier id for app instance instead of a device ID